armory-header-deck/src/components/Header/ArmoryHeader.tsx

```diff
@@ -22,7 +22,7 @@ export const ArmoryHeader = () => {
   const isApplicationView =
     currentState.name.includes('project.application.') || currentState.name.includes('applications.application.');
 
-  const [verticalNavExpanded, setVerticalNavExpanded] = useState<boolean>(false);
+  const [verticalNavExpanded, setVerticalNavExpanded] = useState<boolean>(!CollapsibleSectionStateCache.isSet('verticalNav') || CollapsibleSectionStateCache.isExpanded('verticalNav'));
```

Comment:
initialize the state based on the CollapsibleSectionStateCache to keep it in sync with the recoil state management tool